contracts/ManagerCore.sol

```diff
@@ -178,6 +168,7 @@ contract ManagerCore is Ownable {
      * @param _manager               Address of the manager contract to add
      */
     function addManager(address _manager) external onlyInitialized onlyFactory {
+        require(_manager != address(0), "Zero address submitted.");
```

Comment:
I don't think this is necessary since all calls would come from a trusted contract. We shouldn't need to validate this